### DISCUSSION TITLE
Going back to an older release of the hacktribe firmware ?

### DISCUSSION THREAD

**Discussion by enorrmann** [bangcorrupt/hacktribe#269] [Q&A] [answered]
How can I get a patched version with the firmware prev to the last update ? I'm having some timing issues and want to try if the issue was introduced with the latest one

**Comment by bangcorrupt** [comment, marked as answer]
1. Find the second to last commit where the patch file changed:

```    
git log -n2 patch/hacktribe-2.patch
```

This will print a list of commits:
```

commit a9fa7422bd3501202a11b07ea1c8f9230b1666a0
Author: bangcorrupt <name@domain.com>
Date:   Sat Jul 15 03:38:57 2023 +0100

    mfx

commit 6f4828832eb8b1cc04f93965c143fc4c45cc2ab5
Author: bangcorrupt <name@domain.com>
Date:   Tue Feb 28 09:31:44 2023 +0000

    fix-issue-71

```
(It will be different depending if you are on the `main` or `experimental` branch).

2. Checkout the second commit in the list using its hash:

```
git checkout 6f4828832eb8b1cc04f93965c143fc4c45cc2ab5
```

3. Patch the firmware as you normally would.

If you can describe the issues I can take a look.  If they only happen when using the new MFX it could be an issue with DSP load.  If you are using the experimental branch it could be due to MIDI flooding.  There are some timing issues while the parameter display pop-up is showing, but I think since issue #71 was fixed these are the same as the factory firmware.
